portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,29 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <div id="fullpage">
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section" id="contact">
+        <h1>Check me out!</h1>
+        <div class="socials-grid">
+          <a href="">
+            <img src="https://via.placeholder.com/150/">
```

Comment:
[add multimedia fallbacks](
https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback)
_(For images whose alt attributes would introduce redundancy, and for images whose purpose is purely decorative which you cannot immediately use CSS for, use no alternative text, as in alt="".)_

Reply:
Added multimedia 63747f1